solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -212,6 +214,10 @@ clean() {
     rm -f pack
 }
 
+ctrl_c() {
+    clean 0
```

Comment:
Killing the process with a SIGKILL must return anything distinct from 0.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/6f0a5abafcded5582993be6897cdbf1957a9f2de.